crates/sui-core/src/safe_client.rs

```diff
@@ -437,7 +437,11 @@ impl<C> SafeClient<C> {
     /// error that could be due to this authority.
     /// TODO: Get rid of this. https://github.com/MystenLabs/sui/issues/3740
     pub fn report_client_error(&self, error: &SuiError) {
-        info!(?error, authority =? self.address, "Client error");
+        if matches!(error, SuiError::ValidatorHaltedAtEpochEnd) {
+            debug!(?error, authority =? self.address, "Client error");
```

Comment:
we may need to check substring, some SuiError has a vector of SuiErrors

Reply:
I think that in this case the ValidatorHaltedAtEpochEnd error is always a top-level error.